post_office/test_settings.py

```diff
@@ -1,6 +1,10 @@
-import django
 import os
-from distutils.version import StrictVersion
+import platform
+
+if platform.system() != "Linux":
```

Comment:
Can you modify this to explicitly check for Mac OS? I don't want this to affect Windows users too.

Reply:
I did it in a manner to be extendible. I don't have any Windows here for testing.
Very sad, btw. that we must add platform dependent code into this project :(